src/geometry/Sphere.ts

```diff
@@ -13,15 +13,10 @@ function sphereSignedDistFunc(coord: vec3): number {
     return Math.sqrt(coord[0] * coord[0] + coord[1] * coord[1] + coord[2] * coord[2]) - RADIUS;
 }
 
-// TODO(abhimadan): Create WorkingGeometry here instead
-export function genSphere(): BakedGeometry {
+export function genSphere(): WorkingGeometry {
```

Comment:
Do you think we can make the color a parameter to `genSphere` and then have the `setFill` happen before we return it? and maybe default to some color so we never accidentally call `bake()` with no colors set yet?